share/exchange.yml

```diff
@@ -0,0 +1,1258 @@
+      standard_open: 14h30m
+    standard:
+      daily_close: 21h
+      daily_open: 14h30m
+      daily_settlement: 23h59m59s
+  symbol: NYSE
+  trading_days: weekdays
+  trading_timezone: America/New_York
+NYSE_SPC:
+  currency: USD
+  delay_amount: '1440'
+  display_name: New York Stock Exchange
+  market_times:
+    dst:
+      daily_close: 20h
+      daily_open: 13h30m
+      daily_settlement: 22h59m59s
+    partial_trading:
+      dst_close: 17h
+      dst_open: 13h30m
+      standard_close: 18h
+      standard_open: 14h30m
+    standard:
+      daily_close: 21h
+      daily_open: 14h30m
+      daily_settlement: 23h59m59s
+  symbol: NYSE_SPC
+  trading_days: weekdays
+  trading_timezone: America/New_York
+NZSE:
+  currency: NZD
+  delay_amount: '15'
+  display_name: New Zealand Stock Exchange
+  market_times:
+    dst:
+      daily_close: 6h
+      daily_open: -1h
+      daily_settlement: 9h
+    partial_trading:
+      dst_close: 6h
+      dst_open: -1h
+      standard_close: 5h
+      standard_open: -2h
+    standard:
+      daily_close: 5h
+      daily_open: -2h
+      daily_settlement: 8h
+  symbol: NZSE
+  trading_days: weekdays
+  trading_timezone: Pacific/Auckland
+ODLS:
+  currency: USD
+  delay_amount: '0'
+  display_name: ODL Securities
+  market_times:
+    dst:
+      daily_close: 18h30m
+      daily_open: 7h15m
+      daily_settlement: 23h59m59s
+    partial_trading:
+      dst_close: 11h30m
+      dst_open: 7h15m
+      standard_close: 12h30m
+      standard_open: 8h15m
+    standard:
+      daily_close: 19h30m
+      daily_open: 8h15m
+      daily_settlement: 23h59m59s
+  symbol: ODLS
+  trading_days: weekdays
+  trading_timezone: Europe/London
+OMX:
+  currency: SEK
+  delay_amount: '15'
+  display_name: OMX
+  market_times:
+    dst:
+      daily_close: 15h25m
+      daily_open: 7h
+      daily_settlement: 18h25m
+    partial_trading:
+      dst_close: 11h
+      dst_open: 7h
+      standard_close: 12h
+      standard_open: 8h
+    standard:
+      daily_close: 16h25m
+      daily_open: 8h
+      daily_settlement: 19h25m
+  symbol: OMX
+  trading_days: weekdays
+  trading_timezone: Europe/Stockholm
+OSE:
+  currency: JPY
+  delay_amount: '1440'
+  display_name: Osaka Exchange
+  market_times:
+    partial_trading: ~
+    standard:
+      afternoon_open: 7h30m
+      daily_close: 17h55m
+      daily_open: 0s
+      daily_settlement: 23h59m59s
+      morning_close: 6h15m
+  symbol: OSE
+  trading_days: weekdays
+  trading_timezone: Asia/Tokyo
+OSLO:
+  currency: NOK
+  delay_amount: '15'
+  display_name: OSLO
+  market_times:
+    dst:
+      daily_close: 14h30m
+      daily_open: 07h00m
+      daily_settlement: 17h30m
+    partial_trading:
+      dst_close: 11h
+      dst_open: 7h
+      standard_close: 12h
+      standard_open: 8h
+    standard:
+      daily_close: 15h30m
+      daily_open: 08h00m
+      daily_settlement: 18h30m
+  symbol: OSLO
+  trading_days: weekdays
+  trading_timezone: Europe/Oslo
+RANDOM:
```

Comment:
oh, I see, the Volatility Indices are here. This needs documenting in the README, explaining it is for the binary.com volatility indices. Also the exchange name would be better renamed to VOLATILITY_INDICES